### PR TITLE
removed duplicate handleExportData

### PR DESCRIPTION
**Pull request by RobertBasile-1**
### Verification
 - [ ] PR/Change is verified in local dev environment

### Tests
 - [ ] Unit tests added/updated
 - [ ] Integration tests added/updated

### CODE REVIEW

**Comment by Akram-El-Gaouny** [comment]
Tested the PR locally, when refreshing the page it downloads multiple copies of the exported data.

File: client/src/components/ProjectTable/ProjectTable.js [hunk 1]
```diff
@@ -394,7 +351,7 @@ const ProjectTable = () => {
                 <Button
                   color="primary"
                   //export all data that is currently in the table (ignore pagination, sorting, filtering, etc.)
-                  onClick={handleExportData}
+                  onClick={handleExportData(tableData,columns, csvExporter)}
```
Comment (Akram-El-Gaouny) [inline client/src/components/ProjectTable/ProjectTable.js:354]:
```suggestion
                  onClick={ () => handleExportData(tableData,columns, csvExporter)}
```

File: client/src/components/StudentListingView/StudentTable.js [hunk 2]
```diff
@@ -311,7 +267,7 @@ const StudentTable = () => {
             <Button
               color="primary"
               //export all data that is currently in the table (ignore pagination, sorting, filtering, etc.)
-              onClick={handleExportData}
+              onClick={handleExportData(tableData,columns, csvExporter)}
```
Comment (Akram-El-Gaouny) [inline client/src/components/StudentListingView/StudentTable.js:270]:
```suggestion
                  onClick={ () => handleExportData(tableData,columns, csvExporter)}
```

File: client/src/components/GroupView/GroupTable.js [hunk 3]
```diff
@@ -290,7 +248,7 @@ const GroupTable = () => {
             </Button>
             <Button
               color="primary"
-              onClick={handleExportData}
+              onClick={handleExportData(tableData,columns, csvExporter)}
```
Comment (Akram-El-Gaouny) [inline client/src/components/GroupView/GroupTable.js:251]:
```suggestion
                  onClick={ () => handleExportData(tableData,columns, csvExporter)}
```

**Review by Akram-El-Gaouny — CHANGES_REQUESTED** [review]

**Review by Laith-Grira — APPROVED** [review]

**Review by Akram-El-Gaouny — APPROVED** [review]

**Pull request merged**